packages/itwin/one-click-lca-widget/src/components/ExportModal.tsx

```diff
@@ -94,7 +94,7 @@ const ExportModal = (props: ExportProps) => {
             if (
               currentJobStatus.job?.status === JobStatus.StatusEnum.Succeeded
             ) {
-              setJobLink(currentJobStatus?.job._links?.oneclicklca);
+              setJobLink({href: currentJobStatus?.job._links?.oneclicklca?.href ?? ""});
```

Comment:
If there is no href, the whole link should be null or undefined. Do not set href to an empty string.

Reply:
The thing is, ```oneclicklca``` uses a different ```link``` that is not exported. In the imported ```link``` href is mandatory and in the oneclicklca ```link``` href is optional. This is why this line is the way it is.